config/environments/test.rb

```diff
@@ -40,6 +40,15 @@
   # Tell Active Support which deprecation messages to disallow.
   config.active_support.disallowed_deprecation_warnings = []
 
+  config.time_zone = ActiveSupport::TimeZone.us_zones.sample.name
```

Comment:
Is it possible to include all time zones?

Reply:
ah yes with this ActiveSupport::TimeZone::MAPPING constant. updated in https://github.com/18F/identity-pki/pull/414/commits/b6ab859f2ae9338ed65c391b63f58dafb1cbec35